frontend/packages/kubevirt-plugin/integration-tests-cypress/support/index.ts

```diff
@@ -28,28 +33,25 @@ declare global {
   }
 }
 
-Cypress.Commands.add('deleteResource', (resource, ignoreNotFound = true) => {
-  const kind = resource.kind === 'NetworkAttachmentDefinition' ? 'net-attach-def' : resource.kind;
-
+Cypress.Commands.add('deleteResource', (kind: string, name: string, namespace?: string) => {
```

Comment:
I thought we had a scenario where we'd get an error if we didn't pass `false` for the `ignore-not-found` flag? I didn't see any instances where we were passing `false`, but I wanted to double check with you.

Reply:
yes, I removed the `ignoreNotFound = true` in the parameters because there isn't any instance pass `false` to this function.
If there is such scenario, let's add it back by then.